_about_pages/history.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: History
+type: about
+---
+
+http://calconnect.org/presentations/CC32%20A%20History%20of%20CalConnect.pdf[An
```

Comment:
All `https://calconnect.org` or `http://calconnect.org` links need to be made into `link:...` that serves from the local site.

Reply:
@ronaldtse Can we get all the resources like pdfs and images from Drupal? Otherwise we have to download them manually.

Reply:
We will have to download them manually.